notes_for_translators.md

```diff
@@ -0,0 +1,102 @@
+# Notes for translators
+
+## ⚠️ Critical Rules to Follow
+
+### 1. CSV Format and Separators
+
+Translation files are in **CSV (Comma-Separated Values)** format. Commas separate the file columns.
+
+**Rule:** Commas separating columns **MUST NOT have spaces** before or after them.
```

Comment:
It's good to not have those spaces but it doesn't matter for CSV format validity